README.md

```diff
@@ -28,7 +28,19 @@ All new collections submitted for inclusion MUST satisfy the [Collection Require
 
 If there are any questions, feel free to ask for clarifications in the discussion described below and/or in the `#ansible-community` [Matrix/IRC channel](https://docs.ansible.com/ansible/latest/community/communication.html#real-time-chat).
 
-1. Start a GitHub Discussion in the [new collection review category](https://github.com/ansible-collections/ansible-inclusion/discussions/new?category=new-collection-reviews).  Please copy the following into your initial message (only after you carefully read and make your collection satisfying the [Collection Requirements](https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst)):
+1. Review one of the collections submitted for inclusion by other developers in the following categories:
+  * [second review needed](https://github.com/ansible-collections/ansible-inclusion/discussions/categories/second-review-needed)
+  * [new collection reviews](https://github.com/ansible-collections/ansible-inclusion/discussions/categories/new-collection-reviews)
+
+This step is **optional** but it can significantly reduces the time for your collection to get in (finally it will depend on your collection's quality) because:
+  * You will gain experience when reviewing someone else's collection. It will allow you to make your collection better before submitting it and will save your and reviewer's time and effort as there will probably be fewer issues to fix.
+  * The Steering Committee will handle your request with a higher priority.
+
+You can also offer your help with reviewing other's requests after you submit your collection.
+
+See the [Review process](#review-process) section to learn more.
+
+2. Start a GitHub Discussion in the [new collection review category](https://github.com/ansible-collections/ansible-inclusion/discussions/new?category=new-collection-reviews).  Please copy the following into your initial message (only after you carefully read and make your collection satisfying the [Collection Requirements](https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst)):
```

Comment:
Do we want the full requirements to be pasted into the message? Should we link to the checklist instead?

Reply:
Makes sense, changed, thanks!